### PR TITLE
Feature/crate tracking

### PR DESCRIPTION
**Pull request by NguyenMinhh**

### CODE REVIEW

File: imports/ui/modules/layout/components/TrackingCreationPage/TrackingCreationForm.jsx [hunk 1]
```diff
@@ -1,68 +1,206 @@
...
+    });
+    Meteor.call('tracking.createTracking', values, (error) => {
+      if (!error) {
+        document.getElementById('tracking_inp_field').value = '';
+      }
+    });
+  }
+
+  changeCustomerHandler = () => {
+    document.getElementById('tracking_inp_field').focus();
```
Comment (hoadinh2010) [inline imports/ui/modules/layout/components/TrackingCreationPage/TrackingCreationForm.jsx]: Using ref instead

File: imports/ui/modules/layout/components/TrackingCreationPage/TrackingCreationForm.jsx [hunk 2]
```diff
@@ -1,68 +1,206 @@
...
-export default reduxForm({
-  form: REDUX_FORM_NAMES.TRACKING_CREATION_FORM,
-})(TrackingCreationForm);
\ No newline at end of file
+export default compose(
+  reduxForm({
+    form: REDUX_FORM_NAMES.TRACKING_CREATION_FORM,
+  }),
+  withTracker(() => {
+    const handleCustomer = Meteor.subscribe('customer.getCustomersForCreateTrackingNumber');
```
Comment (hoadinh2010) [inline imports/ui/modules/layout/components/TrackingCreationPage/TrackingCreationForm.jsx]: Change name of this publication

File: lib/classes/tracking/tracking.js [hunk 3]
```diff
@@ -27,7 +27,13 @@ export default class TrackingWorker {
 
   // Check is exist tracking id
   static isExistTracking({ trackingNumber, flightId }) {
-    const tracking = Trackings.findOne({ trackingNumber, flightId });
```
Comment (hoadinh2010) [inline lib/classes/tracking/tracking.js]: Check the condition when get Trackings data

Comment (NguyenMinhh) [reply lib/classes/tracking/tracking.js]: has changed search method from "find" -> findOne.

File: imports/ui/modules/layout/components/TrackingCreationPage/TrackingCreationForm.jsx (hunk 2 above)
Comment (NguyenMinhh) [reply imports/ui/modules/layout/components/TrackingCreationPage/TrackingCreationForm.jsx]: has changed publication's name -> ```getCustomersForCreateFlight```

File: imports/ui/modules/layout/components/TrackingCreationPage/TrackingCreationForm.jsx (hunk 1 above)
Comment (NguyenMinhh) [reply imports/ui/modules/layout/components/TrackingCreationPage/TrackingCreationForm.jsx]: this problem is pending now, will improve later.

**Pull request merged**